dango/lending/src/query.rs

```diff
@@ -0,0 +1,73 @@
+use {
+    crate::{LIABILITIES, WHITELISTED_DENOMS},
+    dango_types::lending::QueryMsg,
+    grug::{Addr, Bound, Coins, Denom, ImmutableCtx, Json, JsonSerExt, Order, StdResult, Storage},
+};
+
+#[cfg_attr(not(feature = "library"), grug::export)]
+pub fn query(ctx: ImmutableCtx, msg: QueryMsg) -> StdResult<Json> {
+    match msg {
+        QueryMsg::WhitelistedDenoms { limit, start_after } => {
+            let res = query_whitelisted_denoms(ctx.storage, limit, start_after)?;
+            res.to_json_value()
+        },
+        QueryMsg::DebtsOfAccount(addr) => {
+            let res = query_debts(ctx.storage, addr)?;
+            res.to_json_value()
+        },
+        QueryMsg::Liabilities { limit, start_after } => {
+            let res = query_liabilities(ctx.storage, limit, start_after)?;
+            res.to_json_value()
+        },
+    }
+}
+
+/// Queries the whitelisted denoms.
+pub fn query_whitelisted_denoms(
+    storage: &dyn Storage,
+    limit: Option<u32>,
+    start_after: Option<Denom>,
+) -> StdResult<Vec<Denom>> {
+    let iter = WHITELISTED_DENOMS.range(
+        storage,
+        start_after.map(Bound::Exclusive),
+        None,
+        Order::Descending,
+    );
+
+    let res = if let Some(limit) = limit {
+        iter.take(limit as usize).collect::<StdResult<Vec<_>>>()?
+    } else {
+        iter.collect::<StdResult<Vec<_>>>()?
+    };
```

Comment:
In all other contracts we use a `DEFAULT_PAGE_LIMIT` constant of 30 in case no limit is specified. Let's keep this consistent across the codebase.